kibble/models/translations.go

```diff
@@ -0,0 +1,24 @@
+//    Copyright 2018 SHIFT72
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+package models
+
+// { "en-au": { "nav_signin": { "other": "Sign In" } } }
+type Translations map[string]map[string]struct {
```

Comment:
What's the difference between this and `TranslationsV1`?

Reply:
There shouldn't be one. My understanding for these "doubleups" was to avoid referencing the /api package from inside the /models package, I think this is done a few times over the repo in different places. At least I recall that was the reasoning @PeculiarGoat ? Other than that I don't there is any other reason we couldn't use TranslationsV1.

Reply:
We are trying to keep the api package/layer separate from the rest of the code of easier maintainability. There is probably a term for this structure (seperation of concerns?) but it escapes me.

As the apis change and new versions are created having an api -> model process and then a separate model -> render process helps with that.

Reply:
Makes sense for back-compat with different versions of an API all having to be massaged through a single render process. It does raise the question of if we are doing that (haven't looked) and if it's a good idea but that's a question for another time.